presto-iceberg/src/main/java/com/facebook/presto/iceberg/TableType.java

```diff
@@ -22,5 +22,7 @@ public enum TableType
     PARTITIONS,
     FILES,
     PROPERTIES,
-    CHANGELOG
+    CHANGELOG,
+    EQUALITY_DELETES,
+    DATE_WITHOUT_EQUALITY_DELETES
```

Comment:
`DATA_WITHOUT_EQUALITY_DELETES`

Reply:
The table type has a special meaning--it's parsed and can be specified through a special syntax (e.g. `table$data_without_equality_deletes`).  It would be better for this not to be exposed publicly, even in an undocumented way.

At this moment I'm not sure if `IcebergTableHandle` is the best place to put this.  As a compromise, let's explicitly disallow its use in `IcebergTableName#from`, perhaps by adding a flag on this enum which specifies if it's a public or private table type (all existing ones would be public).

Reply:
BTW, you can add a trailing comma to prevent a similar diff on line 25 for future additions.  e.g. 
```suggestion
    DATE_WITHOUT_EQUALITY_DELETES,
```